lib/samples/set_up_location_driven_geotriggers/set_up_location_driven_geotriggers.dart

```diff
@@ -0,0 +1,611 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'dart:async';
+import 'dart:collection';
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/common/common.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/utils/sample_state_support.dart';
+import 'package:flutter/material.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class SetUpLocationDrivenGeotriggers extends StatefulWidget {
+  const SetUpLocationDrivenGeotriggers({super.key});
+
+  @override
+  State<SetUpLocationDrivenGeotriggers> createState() =>
+      _SetUpLocationDrivenGeotriggersState();
+}
+
+class _SetUpLocationDrivenGeotriggersState
+    extends State<SetUpLocationDrivenGeotriggers> with SampleStateSupport {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+  // The simulated location data source this sample will use
```

Comment:
```suggestion
  // The simulated location data source this sample will use.
```
This and a few other places missing end periods for tidiness